go-controller/pkg/ovn/namespace.go

```diff
@@ -218,7 +241,68 @@ func (oc *Controller) updateNamespace(old, newer *kapi.Namespace) {
 	}
 	defer nsInfo.Unlock()
 
-	annotation := newer.Annotations[hotypes.HybridOverlayExternalGw]
+	var annotation, oldAnnotation string
+	annotation = newer.Annotations[routingExternalGWsAnnotation]
+	oldAnnotation = old.Annotations[routingExternalGWsAnnotation]
+	if annotation != oldAnnotation {
+		var stderr string
+		var err error
+		for podIP, gwToGr := range nsInfo.podExternalRoutes {
+			var mask string
+			for gw, gr := range gwToGr {
+				if utilnet.IsIPv6(net.ParseIP(podIP)) {
```

Comment:
you defined the function getIPFullMask() why not use it here?

Reply:
oops, thx